bin/eslint.js

```diff
@@ -159,6 +159,8 @@ runWithTiming("engineConfig", function () {
     if (userConfig.debug) {
       debug = true;
     }
+
+    cli = new CLIEngine(options);
```

Comment:
This is currently inside if the `if config.json exists` block, I think, which I'm not sure is what you want. While the missing config.json shouldn't actually happen, we do normally try to have sane defaults if it's missing, don't we?

I think the simple change, if you agree with that reasoning, is just to move this line down one line, outside of the `if` block.